tools/docker/compose

```diff
@@ -91,6 +95,19 @@ case "$1" in
   dev)
     $dev ${@:2}
     ;;
+
+  eth:clean)
+    $base rm --stop --force devnet
+    $base up -d devnet
+    ;;
+  node:clean)
+    node_running=`docker ps -q --no-trunc | grep $(docker-compose ps -q node)`
+    if [ $node_running ]; then $base stop node; fi
+    $base exec node-db dropdb -U postgres node_dev
```

Comment:
Is this method better than dropping the volume the database is attached to?

Reply:
I would have similar concerns as in the comment below.. we'd have to delete the data as root. Also, we need to re-create the db, can't just drop it. The chainlink node will migrate an empty DB but it won't create one if none exists. So I think that is best done using the createdb tool inside the container.